scripts/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -o pipefail
+
+xcodebuild clean build \
```

Comment:
We added both build scripts, but using only `build_app` in actions. Not sure why?

Reply:
Yeah, I added it initially, before I had test script, which will need to build the framework anyways. So I guess I could remove that